news/2018-01-20-beatmap-spotlights-december-2017.md

```diff
@@ -0,0 +1,152 @@
+---
+layout: post
+title: "Beatmap Spotlights: December 2017"
+date: 2018-01-20 09:00:00 +0000
+---
+
+It seems like we still have a few unopened presents from last year - the December Spotlights are out, bundled with a surprise announcement! Whether you are a player or mapper, make sure to drop by and check it out!
+
+
+![](https://assets.ppy.sh/media/generic-header.png)
+
+The beatmap charts for the December Spotlights can be found [here](https://osu.ppy.sh/p/chart?ch=MONTH1801) Their respective packs are available at the [Beatmap Pack Listing](https://osu.ppy.sh/p/packlist?t=r) for easier access. **The December 2017 Spotlights rankings will end on 15th February 2018.**
+
+## Past Spotlights results
+
+First of all, congratulations to the winners of the November 2017 Spotlights. They receive one month of supporter status for their efforts!
++ **osu!:** [Toy](https://osu.ppy.sh/users/2757689)
++ **osu!taiko:** [iceOC](https://osu.ppy.sh/users/5482401)
++ **osu!catch:** [monstratorfull](https://osu.ppy.sh/users/1646397)
++ **osu!mania:** [\_SkyFall](https://osu.ppy.sh/users/1872276)
+
+# Announcing Spotlights Bounty
+
+I'm glad to announce a new type of competition which will run simultaneous with the monthly other Spotlights. We place a bounty on a theme which lacks mapsets and then we ask you as a mapper to map a mapset and rank it until the deadline. After that we choose the bests from this pool of mapsets and announce it.
+
+## How can you enter?
+
++ Map a mapset for any Loki/Thaehan song. It doesn't matter if its not listed [here](https://osu.ppy.sh/beatmaps/artists/7) feel free to check out his other works at [bandcamp](https://loki2.bandcamp.com/) too.
++ The mapset can be in any mode (osu!, osu!taiko, osu!catch, osu!mania).
++ You must map **at least three** (3) difficulties in an appropriate spread as per the Ranking Criteria, either ENH (easy, normal and hard) or NHI (normal, hard, insane).
++ Use SB01 as a tag for easier find.
++ Do not submit mapsets directly to us, instead ask the Beatmap Nominators for help!
++ Your mapset must be ranked before the 15th of March!
+
+## What are the Prizes?
+
++ **1st place:** 3 months of osu!supporter and a shiny new badge, "Bounty Hunter: Thaehan"
++ **2nd place:** 2 months of osu!supporter
++ **3rd place:** 1 month of osu!supporter
++ The prizes are awarded per game mode
+
+## What are the Prizes?
```

Comment:
This should probably have a different title, "Bounty Points system" or something
Also, since this is new, the introduction of the system should be before the prizes so it's clear what they are

Reply:
uh, reread this section of the article because it's just repeating itself lol

Reply:
I know, this is the part where I will need @Ephemeralis to agree which way to go. Two type of prizes listed for that reason.